CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
 
 ## Unreleased 3.0.0 (Breaking release)
 
-See the [versioning documentation for how to update this
-changelog](./docs/contributing/versioning.md#updating-changelog).
+## Breaking changes
+You must make the following changes when you migrate to this release, or your service may break.
 
-- Renamed css class names `js-character-count` and `js-header-toggle`
+### Update file paths, attributes and class names
+To make sure your project's files do not conflict with GOV.UK Frontend, we've moved our package files into a directory called `govuk`.
```

Comment:
I don't know whether this is important enough to warrant a change, but I think our motivation for this was more to avoid GOV.UK Frontend conflicting with the users code, than it is to defend our code against theirs.

Part of me thinks it's nit-picking, and part of me thinks it's actually quite important as it frames the change in a very different light ('we're doing this to make _our_ lives easier, but you have to do some work as a result' vs 'we've made this change because it helps you')…

I know we want to get this merged, so happy to leave it as-is, but might be worth spending 30 seconds thinking about.

Reply:
That seems fair to me - so reversing it to say "To make sure GOV.UK Frontend's files do not conflict with your project's files..." would work?

Reply:
I think so, yep 👍

Reply:
Thanks - updated in https://github.com/alphagov/govuk-frontend/pull/1492/commits/217e12a762789d8040c56dc7831cebc0c9282801